apps/backend/src/modules/reviews/review.repository.ts

```diff
@@ -15,23 +18,64 @@ class ReviewRepository implements Repository {
 
 		const review = await this.reviewModel
 			.query()
-			.insert({
-				content,
-				likesCount,
-				poiId,
-				routeId,
-				userId,
-			})
+			.insert({ content, likesCount, poiId, routeId, userId })
 			.returning("*")
 			.execute();
 
 		return ReviewEntity.initialize(review);
 	}
 
-	public async findAll(): Promise<ReviewEntity[]> {
-		const reviews = await this.reviewModel.query().execute();
+	public async findAll(
+		options: null | ReviewGetAllSearchQuery,
+	): Promise<ReviewEntity[]> {
+		const { routeId } = options ?? {};
+
+		const queryBuilder = this.reviewModel
+			.query()
+			.withGraphJoined("[user(selectUser).avatar(selectAvatar)]")
+			.modifiers({
+				selectAvatar(builder) {
+					builder.select("files.url as url");
+				},
+				selectUser(builder) {
+					builder.select(
+						"users.id",
+						"users.first_name as firstName",
+						"users.last_name as lastName",
+					);
+				},
+			})
+			.select(
+				"reviews.id as id",
+				"reviews.content as content",
+				"reviews.likes_count as likesCount",
+				"reviews.poi_id as poiId",
+				"reviews.route_id as routeId",
+			)
+			.orderBy("reviews.createdAt", SortingOrder.DESC);
+
+		if (routeId) {
+			queryBuilder.where("reviews.route_id", routeId);
+		}
+
+		const rows = await queryBuilder.execute();
 
-		return reviews.map((review) => ReviewEntity.initialize(review));
+		return rows.map((r) =>
```

Comment:
let's use more clear naming
```suggestion
		const reviews = await queryBuilder.execute();

		return reviews.map((review) =>
```